glossary.tex

```diff
@@ -823,10 +770,8 @@
 {
   name=special capability register,
   description={Special capability registers have special architectural
-    meanings, and include the \gls{program-counter capability}, the
-    \gls{default data capability}, the \gls{exception program-counter
-    capability}, the \gls{kernel code capability}, and the \gls{kernel data
-    capability}.
+    meanings, and include the \gls{default data capability} as well
+    as additional architecture-specific capability registers.
     Not all registers are accessible at all times; for example, some may be
     available only in certain rings, or when \PCC{} has the
     Access\_System\_Registers permission set}
```

Comment:
I think there is a macro for `Access\_System\_Registers`?

Reply:
Yes, I've used it here and in a few other places that spelled it out.